source/codegen/metadata/nidcpower/attributes.py

```diff
@@ -0,0 +1,1759 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DCPower API metadata version 20.7.0d8
+attributes = {
+    1050003: {
+        'access': 'read-write',
+        'channel_based': False,
+        'documentation': {
+            'description': '\nSpecifies whether NI-DCPower queries the device status after each operation.\nQuerying the device status is useful for debugging. After you validate your program, you can set this  attribute to VI_FALSE to disable status checking and maximize performance.\nNI-DCPower ignores status checking for particular attributes regardless of the setting of this attribute.\nUse the niDCPower_InitializeWithChannels function to override this value.\nDefault Value: VI_TRUE\n'
```

Comment:
We don't use the `documentation` and `lv_property` values. I realize we forgot to remove them from NI-SCOPE and NI-SWITCH, but consider removing them from here.

Reply:
@jonlugoatx, Done removing these two values from attributes.py file.